airbrakes/mock/mock_logger.py

```diff
@@ -0,0 +1,22 @@
+"""Mock Logger class for testing purposes. Currently only used to delete the log file."""
```

Comment:
I like this class a lot

Reply:
what's there to like tho

Reply:
just a simple and clean way to implement it